picasso-compose/src/main/java/com/squareup/picasso3/compose/PicassoPainter.kt

```diff
@@ -48,12 +51,19 @@ internal class PicassoPainter(
   private val onError: ((Exception) -> Unit)? = null
 ) : Painter(), RememberObserver, DrawableTarget {
 
+  private var lastRequestCreator: RequestCreator? by mutableStateOf(null)
+  private val requestCreator: State<RequestCreator> = derivedStateOf { request(picasso) }
```

Comment:
Nit: Just for consistency with the other states here, this could use property delegation as well.